Cargo.lock

```diff
@@ -776,9 +801,9 @@ checksum = "8a9ee70c43aaf417c914396645a0fa852624801b24ebb7ae78fe8272889ac888"
 
 [[package]]
 name = "hashbrown"
-version = "0.13.2"
+version = "0.13.1"
```

Comment:
Why is this a downgrade? Was 0.13.2 yanked?

Reply:
No it's still there. https://crates.io/crates/hashbrown/0.13.2

Reply:
Nope. This is a transient dependency though, so one of its downstream dependencies might have added an upper bound. I'll have a look later.

Reply:
As I expected: this is because `hashbrown 0.13.1` is pinned specifically by `metrics-util` (its only user): https://crates.io/crates/metrics-util/0.15.1/dependencies

Reply:
Aha, it was pinned to avoid an MSRV bump: https://github.com/metrics-rs/metrics/commit/9d141d1484ecedd3c75528de6490e3fe770f0c50